ci-operator/step-registry/firewatch/report-issues/v1/firewatch-report-issues-v1-ref.yaml

```diff
@@ -0,0 +1,32 @@
+ref:
+  as: firewatch-report-issues-v1
```

Comment:
How will `v1` be used ?

Reply:
It most likely won't be, but it can be. That is there as a last resort in the events that we need to roll back or someone doesn't want to use the new version